extras/docker/fromsource/heketi-start.sh

```diff
@@ -28,6 +28,12 @@ if [[ -d "${HEKETI_PATH}" ]]; then
                 fi
                 ((check+=1))
             done
+        else
+            # database or heketidbstorage do not exist yet, maybe we need to
+            # populate it?
+            if [ -e "${HEKETI_TOPOLOGY_FILE}" ]; then
+                NEED_TOPOLOGY_LOAD=1
+            fi
```

Comment:
Why does this block appear in two places? Can we consolidate this in one place? (for DRY reasons)

Reply:
This is where the directory for the db exists, the other one does not have the directory yet. I could probably move it out of this if/else completely and add a single "if heketi.db does not exist".

Reply:
I think that would be cleaner.

Reply:
This has been done.